openstack/v20.0.0-alpha1/release.yaml

```diff
@@ -27,26 +32,7 @@ spec:
   - name: app-operator
     releaseOperatorDeploy: false
     version: 5.2.0
-  - name: cluster-api
```

Comment:
We have these apps in the openstack-app-collection so we shouldn't have them in the release.